### DISCUSSION TITLE
I didn't do anything with the project, just added "bevy" to Cargo.toml.  And I get an error:

### DISCUSSION THREAD

**Discussion by IOSHED** [bevyengine/bevy#5244] [Q&A] [answered]
PS C:\Users\Admin\NEW> cargo run                                      
   Compiling bevy_dylib v0.7.0
error: linking with `link.exe` failed: exit code: 1189                                                                                                                                 
  |
  = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX64\\x64\\link.exe" "/DEF:C:\\Users\\Admin\\AppData\\Local\\Temp\\
rustcYZ6iQD\\lib.def" "/NOLOGO" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\bevy_dylib-6309eb05ed6be4c5.bevy_dylib.a4d38921-cgu.0.rcgu.o" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\
bevy_dylib-6309eb05ed6be4c5.dfxpxtlwn4kteyy.rcgu.rmeta" "/LIBPATH:C:\\Users\\Admin\\NEW\\target\\debug\\deps" "/LIBPATH:C:\\Users\\Admin\\.cargo\\registry\\src\\github.com-1ecc6299db9e
c823\\windows_x86_64_msvc-0.36.1\\lib" "/LIBPATH:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "C:\\Users\\Admin\\NEW
\\target\\debug\\deps\\libbevy_internal-536af8c2aa73c7eb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_winit-5f08c58d1910be7a.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\
deps\\libwinit-b317cef8902633d3.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_ui-5c3a60b05acf9da7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libstretch-bc0a60a05b1
c0b47.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_text-9c5ade1e084c0d31.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libglyph_brush_layout-71da528baf7c6cf8.rlib" "
C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libxi_unicode-da9519fc5d58d989.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libapprox-93f6e08b6ec39787.rlib" "C:\\Users\\Admin\\NEW\\t
arget\\debug\\deps\\libab_glyph-91f1363082add4b6.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libowned_ttf_parser-56283a9127ddee50.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\de
ps\\libttf_parser-5ec106fad91a5a04.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libab_glyph_rasterizer-07c6add73a7b8f67.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy
_sprite-d23da41b1e488791.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librectangle_pack-5b85f86787e5d672.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libguillotiere-5e31c63
d02de9e87.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libsvg_fmt-8131b4711b5866d7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libeuclid-49daf831dc89548f.rlib" "C:\\Users\
\Admin\\NEW\\target\\debug\\deps\\libbevy_gltf-fd7890ead4da35aa.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libpercent_encoding-3b62996f9d5d32ba.rlib" "C:\\Users\\Admin\\NEW\\ta
rget\\debug\\deps\\libgltf-b762b9618beaed00.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libgltf_json-9ef7d430c26aa631.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libserde
_json-591678546fc8ca29.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libryu-2c443bb695be2c2d.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libitoa-262fc6ced709aa50.rlib" "C:\
\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_pbr-ec17853dbb5dedce.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_gilrs-f21c292621b5c865.rlib" "C:\\Users\\Admin\\NEW\\ta
rget\\debug\\deps\\libgilrs-ff14066cce1b3ad8.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libvec_map-c05b3de97df682fb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libfnv-d7
b83e8424e5a8a9.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libgilrs_core-0aa27204e8e9db51.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librusty_xinput-a62284a7162d568d.rli
b" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_core_pipeline-4c408813521d5bd0.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_render-e1869e60694c392f.rlib" "C:\\Use
rs\\Admin\\NEW\\target\\debug\\deps\\libhex-6899faf9657acef0.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libimage-7cf8aeab0b5bfa83.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\d
eps\\libcolor_quant-f18eb79aa86c7f56.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libnum_iter-2635a8c0b8a34990.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libpng-81cde5a0d
0474076.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libminiz_oxide-618c3096a3819db9.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcrc32fast-ff99aaca4289bc6c.rlib" "C:\\U
sers\\Admin\\NEW\\target\\debug\\deps\\libdeflate-4e619faa5ad73c51.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libadler32-f39f6448f6b6416c.rlib" "C:\\Users\\Admin\\NEW\\target\\
debug\\deps\\libnum_rational-93a62a3edb1ca822.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libnum_integer-d7513a8499a962c8.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libs
coped_threadpool-1275b91c50f786ed.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_crevice-c0a7c581bba814de.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libhexasphere-f
531c7b24d39596f.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libglam-af2eac175cffad22.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libwgpu-6f78ead57e8003f4.rlib" "C:\\Users
\\Admin\\NEW\\target\\debug\\deps\\libwgpu_core-1810d3a79aae5d88.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcopyless-b74072ea36228a90.rlib" "C:\\Users\\Admin\\NEW\\target\\d
ebug\\deps\\libwgpu_hal-f6780911f700e409.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librenderdoc_sys-9a4ffda09f548645.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libgpu_
descriptor-964788498bded78c.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libgpu_descriptor_types-a64556ad472ea210.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libd3d12-5f17
c09926f5e4d8.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libprofiling-a7a9caf5e1fa3ecd.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libgpu_alloc-445ab9cd855c7a54.rlib" "C:
\\Users\\Admin\\NEW\\target\\debug\\deps\\libgpu_alloc_types-dc3602f384e6d063.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libinplace_it-c2d7f4201d7d8883.rlib" "C:\\Users\\Admin\
\NEW\\target\\debug\\deps\\libash-efc51fac883cb0f3.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\liblibloading-8f2ef4e2b7de397e.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\
libnaga-00f1dd44cc2b66c4.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librustc_hash-2d402d6885922b16.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcodespan_reporting-948a
788887687968.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libunicode_width-455c5b87eb401af6.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libtermcolor-5ed9c95666178353.rlib"
 "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libhexf_parse-78e5951f895f25b4.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libpetgraph-bef9ed2316e4df80.rlib" "C:\\Users\\Admin\\NE
W\\target\\debug\\deps\\libpp_rs-ac8f91ad4edc150d.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libunicode_xid-f72c43d39c2c0aa8.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\
libspirv-da2cf62aeb1fec6e.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libnum_traits-03bb444c2ea92997.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libindexmap-24c90507a438d
99a.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libhashbrown-770c62c2a10cb3a4.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librange_alloc-41ee0c3510f84c3f.rlib" "C:\\Users
\\Admin\\NEW\\target\\debug\\deps\\libbit_set-9322369a45fbe340.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbit_vec-21e3bb326bdb3909.rlib" "C:\\Users\\Admin\\NEW\\target\\debu
g\\deps\\libwgpu_types-827326b718ce62ab.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libarrayvec-1b368d5b292b24dd.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_audio
-a938e6e60732f5cb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librodio-f466dba7bad7d5cb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\liblewton-8be8208096035a07.rlib" "C:\\
Users\\Admin\\NEW\\target\\debug\\deps\\libogg-acf9033d2fe2e3e1.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libtinyvec-d1b4d7a4266250d6.rlib" "C:\\Users\\Admin\\NEW\\target\\deb
ug\\deps\\libtinyvec_macros-49b3461bc1e170ff.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcpal-484681b03ace6a2d.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_anim
ation-e8a8dca100c50750.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_window-0d49fc0b5a38f4ef.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libraw_window_handle-5555f4
d085269a60.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcty-7f28babfc96037f7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_transform-9507c6ff546fe7db.rlib" "C:\\U
sers\\Admin\\NEW\\target\\debug\\deps\\libbevy_scene-efa19f1009ec3dd7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libron-ae0b5465895c584b.rlib" "C:\\Users\\Admin\\NEW\\target\\d
ebug\\deps\\libbase64-343b16dd28cc142e.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbitflags-af632af1c4fe8fff.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_hierar
chy-a2c1def6af2ab2a9.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_input-ca764a78544bec32.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_asset-00c025a4e4192664
.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librand-5656bdd30a46f484.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librand_chacha-f43a3783a9f9dc64.rlib" "C:\\Users\\Admin\
\NEW\\target\\debug\\deps\\libppv_lite86-df214066711a8c02.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\librand_core-1f0c0fc2b20ff2a1.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\
deps\\libnotify-7b7db34e4d964c25.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libwalkdir-d04b3fc836e7faad.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libsame_file-a3303060
1283a717.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libwinapi_util-864d7e775d674283.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libfiletime-c7174570426c7912.rlib" "C:\\U
sers\\Admin\\NEW\\target\\debug\\deps\\libwindows_sys-6c63bb724249d762.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_diagnostic-6b59752c38db902a.rlib" "C:\\Users\\Admin\\N
EW\\target\\debug\\deps\\libbevy_core-88fffa9c9726a297.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_math-78d0461ce0e77988.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\dep
s\\libcrossbeam_channel-1c94123f52f7c093.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcrossbeam_utils-d679fe1ffafb6812.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbe
vy_log-21e8db33199bcb2f.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libtracing_subscriber-d70f35c9e5d80a3d.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libsharded_slab-f12
c15c376a159de.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libmatchers-b07118a3c5bb2ac7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libregex_automata-b3f406955ea7dc09.rlib
" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libregex-0a7b93d290dd0844.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libaho_corasick-a8dc03cd499d0f68.rlib" "C:\\Users\\Admin\\NE
W\\target\\debug\\deps\\libregex_syntax-4957f9cfa535466c.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libansi_term-89cad9b4d92a9410.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\d
eps\\libthread_local-1eae52507ce43a0f.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libtracing_log-364a37236a4be1cb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\liblog-16a47
46985648c3b.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\liblazy_static-abd29bacf6e0c750.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libanyhow-cdd6a9a325ef1b97.rlib" "C:\\
Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_app-9b161310c53e7812.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_ecs-6a55405e627ce73d.rlib" "C:\\Users\\Admin\\NEW\\targe
t\\debug\\deps\\libfxhash-650a7fa27a69ec14.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbyteorder-f7ea22e092bd18f5.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_t
asks-d0d67ef9d646eae6.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libasync_channel-99417595006ba679.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libasync_executor-bd8bfcda
58a70b0c.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libslab-7896b571d5ab8f8f.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libconcurrent_queue-7265feb9eee5742c.rlib" "C:\\
Users\\Admin\\NEW\\target\\debug\\deps\\libcache_padded-213b71cbeaa2a0dc.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libasync_task-59b386c3ba1b8d80.rlib" "C:\\Users\\Admin\\NEW\
\target\\debug\\deps\\libnum_cpus-14e59cc5c4d38bd1.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libevent_listener-66d112198c7c05bd.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\de
ps\\libfutures_lite-e6de6ba0fbc02125.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libmemchr-d2a89dee6865bb8b.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libfastrand-2eff78
1a085d3e0a.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libfutures_io-c6500f34a8719ea2.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libfutures_core-ee09d373bc66e3ff.rlib" "
C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libwaker_fn-7d9cfa5d48a3b803.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libparking-2e9981e055d4a674.rlib" "C:\\Users\\Admin\\NEW\\ta
rget\\debug\\deps\\libfixedbitset-5cdf30af05fc9f4b.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_reflect-d69a64ccd98f1641.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps
\\liberased_serde-439edfc2885c0066.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libglam-695fde9173885d7f.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libmint-90cd8ac099c8ab
6c.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbytemuck-b9ae6426712ce9d3.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libparking_lot-3f70fcbbffc06488.rlib" "C:\\Users\\
Admin\\NEW\\target\\debug\\deps\\libparking_lot_core-90a6197b91712c6c.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libwinapi-d993e2233020d19c.rlib" "C:\\Users\\Admin\\NEW\\target
\\debug\\deps\\libsmallvec-2121130229b3c9f9.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\liblock_api-0c63ff89b67ae7b9.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libscopeg
uard-a3146c178c010143.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libdowncast_rs-900f79c5b7024e03.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libthiserror-f01380d95663f28
a.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libbevy_utils-7b88ad0b7051edf7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libuuid-f96ce823f758f6fc.rlib" "C:\\Users\\Admin\
\NEW\\target\\debug\\deps\\libtracing-0059ea8f8eb035f5.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libpin_project_lite-b2ca87158b4c4ddd.rlib" "C:\\Users\\Admin\\NEW\\target\\deb
ug\\deps\\libtracing_core-b4ee41627f6b8bf7.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libinstant-35753ab563b11393.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libhashbrow
n-67f48e7ae43a9179.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libserde-42f4af9d21622c49.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libahash-98afce9d971f0698.rlib" "C:\\
Users\\Admin\\NEW\\target\\debug\\deps\\libgetrandom-1883e4ee8e8472eb.rlib" "C:\\Users\\Admin\\NEW\\target\\debug\\deps\\libcfg_if-a21575eac01a5ea9.rlib" "C:\\Users\\Admin\\NEW\\target
\\debug\\deps\\libonce_cell-1225f0f6fac3dbe4.rlib" "/LIBPATH:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "std-c2061
a8c4363ad39.dll.lib" "C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-c9e4ee006e473c58.rlib" "wind
ows.lib" "advapi32.lib" "cfgmgr32.lib" "comctl32.lib" "d3d12.lib" "d3dcompiler.lib" "dwmapi.lib" "dxgi.lib" "gdi32.lib" "imm32.lib" "kernel32.lib" "mmdevapi.lib" "msimg32.lib" "ole32.l
ib" "opengl32.lib" "shcore.lib" "shell32.lib" "shlwapi.lib" "synchronization.lib" "user32.lib" "uxtheme.lib" "winmm.lib" "winspool.lib" "xinput.lib" "bcrypt.lib" "kernel32.lib" "ws2_32
.lib" "bcrypt.lib" "advapi32.lib" "userenv.lib" "kernel32.lib" "msvcrt.lib" "/NXCOMPAT" "/LIBPATH:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\x8
6_64-pc-windows-msvc\\lib" "/OUT:C:\\Users\\Admin\\NEW\\target\\debug\\deps\\bevy_dylib-6309eb05ed6be4c5.dll" "/OPT:REF,NOICF" "/DLL" "/IMPLIB:C:\\Users\\Admin\\NEW\\target\\debug\\dep
s\\bevy_dylib-6309eb05ed6be4c5.dll.lib" "/DEBUG" "/NATVIS:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\
\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\Admin\\.rustup\\toolchains\\stable-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis"
  = note: Non-UTF-8 output: LINK : fatal error LNK1189: \xaf\xe0\xa5\xa2\xeb\xe8\xa5\xad \xaf\xe0\xa5\xa4\xa5\xab \xa1\xa8\xa1\xab\xa8\xae\xe2\xa5\xaa\xa8 \xa2 65535 \xae\xa1\xea\xa5\xaa\xe2\xae\xa2\r\n

error: could not compile `bevy_dylib` due to previous error

**Comment by Nilirad** [comment, marked as answer]
Looks like cargo is trying to dynamically link bevy to your crate, which has issues on Windows. That's strange, since you have to explicitly enable that feature.
